cmd/submit.go

```diff
@@ -261,6 +261,10 @@ func (s *submitCmdContext) submit(metadata *workspace.ExerciseMetadata, docs []w
 		return fmt.Errorf(jsonErrBody.Error.Message)
 	}
 
+	if resp.StatusCode < 200 || resp.StatusCode > 299 {
+		return fmt.Errorf("submission unsuccessful (%s)", resp.Status)
```

Comment:
I'm guessing we might want to say more here. What do we want this to say?

Reply:
The API should respond with `{"error": {"type": "some_value", "message": "A human-readable message"}}` in the case of an error. Let's use the `message` if it has something available in it, otherwise let's use the default that you have here.

Reply:
Actually, you wrote code that fixed it. I've extracted that here:
https://github.com/exercism/cli/pull/814